styles/components/Layout.module.scss

```diff
@@ -86,7 +86,7 @@ Layout: Content
   min-height: calc(100vh - 150px);
 
   .layout__content_sizer {
-    max-width: 1200px;
+    max-width: 1350px;
```

Comment:
Does this have any effect on the designs for the other pages?

Reply:
It just increases the maximum width of content in each page. I think it does change the layout of the pages, however. I needed it here since the content on this page is supposed to be a little larger than the originally defined 1200px

Reply:
We should check with Julia on this, just to check if she intended for the width to be different on just one page.